### PR TITLE
Add Bounties ETHGlobal Brussels

### PR DESCRIPTION
**Pull request by biancabuzea200**
Add Bounties for ETHGlobal Brussels and resources

### CODE REVIEW

File: docs/hackathons/eth-global-brussels-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+The prize will go to the top three teams with the most innovative integration of Scribe, Chronicle's new Schnorr Oracle. You can deploy your contracts on any of the following chains:
+
+- Ethereum Sepolia
+- Arbitrum Sepolia
+- Base Sepolia
+- Polygon zkEVM Sepolia
+- Gnosis Chain Mainnet
+- Scroll Sepolia
+- zkSync Sepolia 
+- Polygon zkEVM Sepolia
```
Comment (pmerkleplant) [inline docs/hackathons/eth-global-brussels-hackathon.md:33]: Polygon zkEVM mentioned 2 times

File: docs/hackathons/eth-global-brussels-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+- Mantle Sepolia 
+
+Solutions must integrate the Chronicle Oracle smart contracts. The innovation should stem from the use case of the Oracle, i.e., The idea or utility of the project.
```
Comment (pmerkleplant) [inline docs/hackathons/eth-global-brussels-hackathon.md:36]: > [...] of the Oracle, i.e., The idea

The should be lowercase

File: docs/hackathons/eth-global-brussels-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+
+You can compete for both tracks, however you will get the bounty only for one of them. 
+
+### 📜 Qualification rules:
+
+- Must utilize the Oracle smart contract on one of the supported blockchains.
+- Each project must use Chronicle Oracles in some form to make a state change on a blockchain, otherwise it will not be eligible. This means that a front end simply reading from Chronicle Oracles doesn't count.
```
Comment (pmerkleplant) [inline docs/hackathons/eth-global-brussels-hackathon.md:43]: > must make a state change

nice!

**Review by pmerkleplant — APPROVED** [review]
Two nits, otherwise lgtm.

Also noticed we don't have the `SelfKisser` setup yet for every chain. Created a ticket, see https://app.shortcut.com/chronicle-labs/story/6762/deploy-selfkisser-and-verify-all-stage-oracles-are-self-kissable.

Once the ticket is done we should update this docs to add the `SelfKisser` addresses.

File: docs/hackathons/eth-global-brussels-hackathon.md (hunk 1 above)
Comment (biancabuzea200) [reply docs/hackathons/eth-global-brussels-hackathon.md:33]: Thank you, Pascal. I wrote to Alex this morning to try to squeeze in in the engineering pipeline the deployments for the SelfKisser on Arbitrum, Base, and Optimism Testnet.

**Review by pmerkleplant — APPROVED** [review]

**Pull request merged**